pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -27,17 +27,23 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/wait"
+	corelisters "k8s.io/client-go/listers/core/v1"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/pkg/controller/volume/attachdetach/cache"
 	"k8s.io/kubernetes/pkg/controller/volume/attachdetach/metrics"
 	"k8s.io/kubernetes/pkg/controller/volume/attachdetach/statusupdater"
+	attachdetachutil "k8s.io/kubernetes/pkg/controller/volume/attachdetach/util"
 	kevents "k8s.io/kubernetes/pkg/kubelet/events"
 	"k8s.io/kubernetes/pkg/util/goroutinemap/exponentialbackoff"
 	"k8s.io/kubernetes/pkg/volume/util"
 	"k8s.io/kubernetes/pkg/volume/util/operationexecutor"
 )
 
+const (
+	OutOfServiceTaint = "out-of-service"
+)
+
```

Comment:
Thinking, if we should move this to types? Or any proper home for this constant?

Reply:
Move it to the following file?
https://github.com/kubernetes/kubernetes/blob/v1.24.0-alpha.3/pkg/util/taints/taints.go#L269